packages/automation-contracts/scheduler/contracts/VestingSchedulerV2.sol

```diff
@@ -93,37 +94,28 @@ contract VestingSchedulerV2 is IVestingSchedulerV2, SuperAppBase {
         _validateAndCreateVestingSchedule(
             ScheduleCreationParams({
                 superToken: superToken,
+                sender: _getSender(bytes("")),
                 receiver: receiver,
-                startDate: startDate,
+                startDate: _normalizeStartDate(startDate),
                 claimValidityDate: claimValidityDate,
                 cliffDate: cliffDate,
                 flowRate: flowRate,
                 cliffAmount: cliffAmount,
                 endDate: endDate,
                 remainderAmount: 0
-            }),
-            bytes("")
+            })
         );
     }
 
     function _validateAndCreateVestingSchedule(
-        ScheduleCreationParams memory params,
-        bytes memory ctx
-    ) private returns (bytes memory newCtx) {
-        newCtx = ctx;
-        address sender = _getSender(ctx);
-
-        // Default to current block timestamp if no start date is provided.
-        if (params.startDate == 0) {
```

Comment:
startDate normalization is moved to a single place and it's expected to pass in correctly here